_includes/templates/nav/_vars-primary-nav.html

```diff
@@ -0,0 +1,41 @@
+
+{#
+  Navigation and Sub-Navigation menu items,
+  used to populate the contents of the site's navigation menus.
+  Format: navigation title {{ string }},
+          sub-navigation title {{ string }},
+          sub-navigation media {{ HTML string }},
+          sub-navigation items {{ array of tuples }}
+#}
+
+{% import "templates/nav/about-us-media.html" as about_us_media %}
+
+{% set nav_tbd =  ('TBD Content', '', '', []) %}
+{% set sub_nav_tbd =  ('', '', 'TBD Content') %}
+{% set nav_items = [
+    nav_tbd,
+    nav_tbd,
+    nav_tbd,
+    ('About', 'About Overview', about_us_media , [
+        (
+            ('/the-bureau/', 'the-bureau', 'The Bureau'),
+            ('/budget/', 'budget', 'Budget and Strategy'),
+            ('/offices/payments-to-harmed-consumers/',
+             'payments-to-harmed-consumers',
+             'Payments to Harmed<br>Consumers'),
```

Comment:
@anselmbradford , I think it's a bit confusing but that is how it should be styled.